lib/icalendar/util/deserialize.ex

```diff
@@ -254,6 +256,10 @@ defmodule ICalendar.Util.Deserialize do
       else
         Timex.Timezone.Utils.to_olson(timezone)
       end
+      |> case do
+        nil -> @default_timezone
+        tz -> tz
+      end
```

Comment:
If only `Timex.Timezone.Utils.to_olson(timezone)` can be `nil` then:
```suggestion
        Timex.Timezone.Utils.to_olson(timezone) || @default_timezone
      end
```